pyproject.toml

```
@@ -1,3 +1,65 @@
+[build-system]
+requires = [
+    "wheel",
+    "setuptools",
+    "Cython>=3",
+    # taken from scipy (2024/06/28)
+    # numpy requirement for wheel builds for distribution on PyPI - building
+    # against 2.x yields wheels that are also compatible with numpy 1.x at
+    # runtime.
+    # Note that building against numpy 1.x works fine too - users and
+    # redistributors can do this by installing the numpy version they like and
+    # disabling build isolation.
+    "numpy>=2.0.0rc1",
```

nit: 2.0.0 has been released...
```suggestion
    "numpy>=2.0.0",
```